### PR TITLE
PS-5260: xtrabackup-plugin-dir - add definition

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst [hunk 1]
```diff
@@ -657,4 +658,10 @@ Options
 
    This option prints |xtrabackup| version and exits.
 
+.. option:: --xtrabackup-plugin-dir=DIRNAME
+
+   The absolute path of the directory that contains the
+   ``keyring_file`` plugin (:file:`keyring_file.so`). This option is
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst]: Not just keyring_file. xtrabackup shipped with two plugins: keyring_file and keyring_vault. I would not mention keyring-file-data specifically, instead I would give a link to https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html#keyring-vault-plugin and https://dev.mysql.com/doc/refman/5.7/en/keyring-file-plugin.html for option reference

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst (hunk 1 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst]: Ready for review

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst [hunk 2]
```diff
@@ -657,4 +658,16 @@ Options
 
    This option prints |xtrabackup| version and exits.
 
+.. option:: --xtrabackup-plugin-dir=DIRNAME
+
+   The absolute path of the directory that contains the
+   ``keyring_file`` plugin (:file:`keyring_file.so`).
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst]: maybe simply keyring plugin, or keyring plugins?

Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst]: Ready for review

**Review by gl-sergei — APPROVED** [review]

**Pull request merged**